test/index.js

```diff
@@ -57,9 +81,28 @@ describe('index', function() {
   });
 
   describe('openSync()', function() {
+    var readFileSpy;
+    beforeEach(function(done) {
+      sinon.stub(fs, 'watch', function(path, cb) {
+        cb();
+      });
+      readFileSpy = sinon.spy(fs, 'readFileSync');
+      done();
+    });
+    afterEach(function(done) {
+      fs.watch.restore();
+      readFileSpy.restore();
+      done();
+    });
```

Comment:
i can, but i reinit' them after each test case so the count of (calledOnce / calledTwice) will be for each test and not aggregated.